### PR TITLE
metrics-overview: export "total objects" (consumers, connections, etc.)

### PR DESCRIPTION
**Pull request by multani**
#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass

#### Purpose

This adds counts of global objects related to the cluster, such as number of total connections, total channels, total consumers, etc.

#### Known Compatibility Issues

This exports 5 new metrics of an already existing metrics plugin.

### CODE REVIEW

**Comment by oferrigni** [comment]
Can't find any place that this would be tested and the checks present look more thorough than previous features.

**Comment by oferrigni** [comment]
Is it possible to post a screen shot showing the feature working?

**Comment by multani** [comment]
@oferrigni Here you go!

screenshot from 2018-06-15 13-45-09

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+### Changed
```
Comment (majormoses) [inline CHANGELOG.md]: This should be `### Added` but I will fix up post merge prior to release.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-rabbitmq/versions/5.3.0

**Pull request merged**